tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/pcluster.config.update_drain.yaml

```diff
@@ -0,0 +1,103 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    S3Access:
+      - BucketName: {{ bucket_name }}
+Scheduling:
+  SlurmSettings:
+    QueueUpdateStrategy: DRAIN
+  Scheduler: slurm
+  SlurmQueues:
+    - Name: queue1
+      Iam:
+        S3Access:
+          - BucketName: {{ bucket_name }}
+      ComputeResources:
+        - Name: queue1-i1
+          InstanceTypeList:
+            - InstanceType: c5.xlarge
+          MinCount: 1
+          MaxCount: 2
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+    - Name: queue2
+      Iam:
+        S3Access:
+          - BucketName: {{ bucket_name }}
+      ComputeResources:
+        - Name: queue2-i1
+          InstanceTypeList:
+            - InstanceType: c5.xlarge
+          MinCount: 1
+          MaxCount: 2
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+SharedStorage:
+  - MountDir: /raid_dir
+    StorageType: Ebs
+    Name: raid-type1
+    EbsSettings:
+      VolumeType: io1
+      Size: 15
+      Iops: 750
+      Encrypted: false
+      Raid:
+        Type: 0
+        NumberOfVolumes: 5
+  - MountDir: {{ new_ebs_mount_dir }}
+    Name: /ebs1
+    StorageType: Ebs
+    EbsSettings:
+      VolumeType: gp3
+      DeletionPolicy: Delete
+  - MountDir: {{ existing_ebs_mount_dir }}
+    Name: existing_eebs
+    StorageType: Ebs
+    EbsSettings:
+      VolumeType: gp2
+      VolumeId: {{ volume_id }}
+  - MountDir: {{ new_efs_mount_dir }}
+    Name: efs1
+    StorageType: Efs
+    EfsSettings: # Customize EFS settings to have better coverage outside other EFS tests.
+      Encrypted: true
+      PerformanceMode: maxIO
+      ThroughputMode: provisioned
+      ProvisionedThroughput: 200
+  - MountDir: {{ existing_efs_mount_dir }}
+    Name: existing_efs
+    StorageType: Efs
+    EfsSettings:
+      FileSystemId: {{ existing_efs_id }}
+  - MountDir: {{ new_lustre_mount_dir }}
+    Name: fsx1
+    StorageType: FsxLustre
+    FsxLustreSettings:
+      StorageCapacity: 1200
+      ImportPath: s3://{{ bucket_name }}
+      ExportPath: s3://{{ bucket_name }}/export_dir
+      DeploymentType: PERSISTENT_1
+      PerUnitStorageThroughput: 200
+  - MountDir: {{ fsx_lustre_mount_dir }} # the index of mount dirs starts from the second item. Because the first is used for the new FSx.
```

Comment:
I don't get the goal of the comment in this line

Reply:
removed